### PR TITLE
Made the main page responsive

### PR DESCRIPTION
**Pull request by kshitij01042002**
Fixes issue #146

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,9 +16,10 @@
   </head>
   <style id="scrollbar_style"></style>
   <body>
-    <div id="root"></div>
+    <div id="root">
     <script type="module" src="/src/main.jsx"></script>
-    <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js"></script>
-    <script src="/src/script.js" type="module"></script>
+    <!-- <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js"></script>
```
Comment (narayan954) [inline index.html:21]: why did you comment out the whole feature XD

**Comment by narayan954** [comment]
image
image
Issue still persists

**Comment by kshitij01042002** [comment]
image

@narayan954 This works?

**Review by narayan954 — APPROVED** [review]
lgtm!

**Comment by narayan954** [comment]
> image
> 
> @narayan954 This works?

couldn't check with loaded posts as firebase's storage limit exceeded for the day (now I have updated the security rules to strict), rest works fine!

**Pull request merged**